.authors.yml

```
@@ -888,3 +888,13 @@
   num_commits: 1
   first_commit: 2020-12-29 19:51:32
   github: timgates42
+- name: Thomas Donoghue
+  email: tdonoghue.research@gmail.com
+  alternate_emails:
+  - tdonoghue@ucsd.edu
+aliases:
+  - Tom Donoghue
+  - Tom
```

```suggestion
  aliases:
  - Tom Donoghue
  - Tom
```

Could anything better represent modern development than breaking YAML syntax by editing it in a web UI? :disappointed: